repo/packages/N/nexus/0/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "3.0",
+  "name": "nexus",
+  "version": "1.0-3.1.0",
+  "minDcosReleaseVersion" : "1.8",
+  "scm": "https://github.com/sonatype/docker-nexus3",
+  "maintainer": "support@sonatype.org",
+  "website": "http://www.sonatype.org/nexus/",
+  "framework": false,
+  "description": "Nexus is a software repository manager. It supports Bower, Docker, Maven 2, npm, NuGet, PyPI, and Raw site repositories",
```

Comment:
add a link to the `example` at the end of `description`: 
Installation Documentation: https://github.com/dcos/examples/tree/master/1.8/......